src/clib/CMakeLists.txt

```diff
@@ -329,6 +329,12 @@ if (ADIOS_USE_COMPRESSION)
     PUBLIC _SPIO_ADIOS_USE_COMPRESSION)
 endif ()
 
+if (ADIOS_USE_LOSSY_COMPRESSION)
+  message(STATUS "Applying lossy compression to ADIOS BP files to reduce disk usage")
```

Comment:
Can you change this text to omit "to reduce disk usage" ? The more important message to the user is the compression algorithm used by the library (please add info on the default compression algo in the message)